docs/models.yml

```diff
@@ -2041,6 +2074,7 @@ motion_block:
     on_delete: CASCADE
     equal_fields: meeting_id
     restriction_mode: B
+    required: true
```

Comment:
What exactly means the `required: true`? Does it mean at this point, that every motion_block will always have an agenda_item assigned? If yes, than this is wrong. The `agenda_item_id` of a motion_block can be `null`.

Reply:
Not automatically, for each one an agenda_item can be created. But, if an agenda_item was created, this can be deleted afterwards and the related content_object's agenda_item is `null`.